first-analysis-steps/analysisflow.md

```diff
@@ -0,0 +1,114 @@
+# The analysis flow and analysis preservation
+
+{% objectives "Learning Objectives" %}
+
+* Learn about analysis tools
+* Learn about analysis preservation 
+
+{% endobjectives %}
+
+Most LHCb data analyses share similar structure and follow similar procedure pipeline.
+As a user you start developing your analysis from retrieving the data and simulation samples that you need for your research.   
+Giving you tools to get those samples is the main goal of the first analysis steps lessons.  
+
+{% callout "Reminder from the data flow lesson" %}
+
+The output of the last stages of the [LHCb data flow](dataflow), namely Stripping (Run 1 and Run 2), Turbo stream (Run 2), Sprucing (Run 3) or Run 3 HLT2 lines can be accessed by user. 
+Selections made at these last stages are also defined by users. 
+Their output is saved to disk and later is used to produce a data file that will contain only the information that interests users.
+This is done using the software package called DaVinci.
+
+{% endcallout %}
+
+### Getting data files
+
+After preselecting data either in the stripping, sprucing or triggering step, user can produce a ROOT file called _ntuple_, by running the DaVinci package.
+The ntuple is a data file that contains information about events or candidates in the data sample, like mass of a candidate or trigger decision flags.
```

Comment:
I left just now a few suggestions but will have to pause. I will try and comment on the rest either later today or during the week. I see things that clearly need to be corrected - for example an ntuple is _not_ a file, it is rather a data structure typically stored within a (ROOT) file. Maybe worth linking to the ROOT manual for definitions of an ntuple? Else add a definition to our glossary https://github.com/lhcb/glossary and link to that.

Reply:
I opened a PR in glossary. As soon as merged I will link it here.